deno/src/core/client.ts

```diff
@@ -206,6 +206,12 @@ class ApiClient<R extends RawApi> {
         private readonly webhookReplyEnvelope: WebhookReplyEnvelope = {}
     ) {
         this.options = { ...DEFAULT_OPTIONS, ...options }
+        if (this.options.apiRoot.endsWith('/'))
+            throw new Error(
+                `Remove the trailing '/' from the 'apiRoot' option! (Use '${this.options.apiRoot.substr(
+                    this.options.apiRoot.length - 1
+                )}' instead of '${this.options.apiRoot}'.)`
```

Comment:
If the logic already suggests what exactly should be done why not just do it directly without error?

Reply:
1. Being explicit in the user config is better than applying opaque transformations to it.
2. Consistency: https://github.com/grammyjs/grammY/blob/main/deno/src/composer.ts#L385-L392